### PR TITLE
created hero component

### LINKED ISSUE #3: Hero component
#### Screenshot
Screenshot 2020-03-14 at 22 26 57
--------
Screenshot 2020-03-14 at 22 33 25
------

#### Used by
 - All the projects

#### Name
 - Hero

#### Children
 - Icon (optional)

#### Props
 - Title (required)
 - Subtitle (optional)

#### Other
 - We can can have Only title, title and subtite, title and icon, and all of them at the same time.

### PR DESCRIPTION
**Pull request by acis**
Closes #3 

Created Hero component

### CODE REVIEW

File: src/components/hero/hero.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.hero-body {
+  font-family: Titillium Web, sans-serif;
```
Comment (moonflare) [inline src/components/hero/hero.scss]: If you do a rebase from the upstream repo, this PR https://github.com/code4romania/taskforce-fe-components/pull/21 adds these by default. We can also remove it from L17.
```suggestion
```

**Comment by aniri** [comment]
hey @acis ! thanks for your help! could you please fix the merge conflicts and check the comments above, please? :smile:

File: src/components/hero/hero.stories.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Hero } from "./hero";
+
+const title = "Ce pasi ai de urmat";
+const subtitle = "Pentru a te putea ajuta ...";
+const icon = <img src={require("./covid-all.svg")} alt={"foo"} />;
```
Comment (Utwo) [inline src/components/hero/hero.stories.js]: We used this way of importing svgs in this repo: https://github.com/code4romania/taskforce-fe-components/blob/master/src/components/header/header.js#L4  
Please change the import and also add a relevant alt attribute :)

File: .eslintrc [hunk 3]
```diff
@@ -1,7 +1,8 @@
 {
   "parser": "babel-eslint",
   "env": {
-    "browser": true
+    "browser": true,
+    "amd": true
```
Comment (Utwo) [inline .eslintrc]: After changing the import you can remove this

File: src/components/hero/hero.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./hero.scss";
+
+export const Hero = ({ title, subtitle, children }) => {
+  return (
+    <section className="hero">
+      <div className="hero-body">
+        <div className="title">
+          {children ? <span className="hero-icon">{children}</span> : <></>}
```
Comment (Utwo) [inline src/components/hero/hero.js]:
```suggestion
          {children && <span className="hero-icon">{children}</span>}
```

**Comment by surdu** [comment]
Other than what @Utwo mentioned, all looks good to me

**Comment by moonflare** [comment]
Screenshot 2020-03-17 at 00 45 53

I believe in Figma the icon is aligned a bit differently. Can you please double check it?

Also, I think it could be beneficial to have that icon (front the Story) in the component and perhaps disabled by default (with a flag we could toggle), but still be able to pass another icon as we do now. What do you think?

File: src/components/hero/hero.stories.js (hunk 2 above)
Comment (acis) [reply src/components/hero/hero.stories.js]: yep, realised this while implementing the header component  👍

**Pull request merged**